src/components/CompanyHomePage/Tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+import { Col, Row, Tab, Nav } from 'react-bootstrap'
+import MatchCard from './MatchCard'
```

Comment:
I don't think your pr should not include this. It should only add `chatBox` component and nothing else